rust/signer/Makefile

```diff
@@ -1,7 +1,9 @@
-NDK_STANDALONE = $(shell pwd)/../../NDK
 ARCHS_IOS = x86_64-apple-ios armv7-apple-ios armv7s-apple-ios aarch64-apple-ios
 ARCHS_ANDROID = aarch64-linux-android armv7-linux-androideabi i686-linux-android
 LIB=libsigner.a
+API_LEVEL=29
```

Comment:
It would be good to get this value from `./scripts/variables.sh` file. AFAIK I can't do it, or probably we could use shell script instead of MakeFile here.

Reply:
Does it help:

```
$ API_LEVEL="$(grep API_LEVEL scripts/variables.sh |tr -c -d '[:digit:]')"
$ echo $API_LEVEL 
29
```

?

Reply:
thank you! It works!